app/assets/javascripts/components/mentors/_body.js.jsx

```diff
@@ -8,8 +8,9 @@ var Body = React.createClass({
   },
 
   searchMentors(query){
+    query = query.toLowerCase();
     let mentors = this.state.allMentors.filter((mentor) => {
-      return mentor.name.includes(query) || mentor.location.includes(query) || mentor.expertise.includes(query) || mentor.company.includes(query)
+      return mentor.name.toLowerCase().includes(query) || mentor.location.toLowerCase().includes(query) || mentor.expertise.toLowerCase().includes(query) || mentor.company.toLowerCase().includes(query) || mentor.bio.toLowerCase().includes(query)
     });
```

Comment:
I don't love this boolean. What if you concatenate all of the mentor properties into one big string, and check against that. Turn it into two lines.

Also, can't hurt to `toLowerCase()` on your query.